launch.go

```diff
@@ -114,6 +136,21 @@ func launch(api screwdriver.API, buildID string) error {
 	if err != nil {
 		return err
 	}
+	prNumber, err := checkIfPr(j.Name)
```

Comment:
"check" to me doesn't imply returning the PR number. We should revisit this interface.